dss-api.yml

```diff
@@ -588,6 +589,7 @@ paths:
       description: |
         Create a new version of a bundle with a given UUID.  The list of file UUID+versions to be included must be
         provided.
+      x-dss-idempotent: true
```

Comment:
I was under impression that this is not required when using PUT

Reply:
Whoops, need to update the PR.